tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -392,21 +399,51 @@ def _execute_test_suite(self, suite: TestSuiteDescription) -> bool:
                         agent_test_logger.info("\t%s", r)
                     agent_test_logger.info("")
 
-                    if len(failed) == 0:
```

Comment:
this is no longer needed. we were reporting a list of test names for failures, but now each failure would be reported individually